.github/workflows/create-release.yml

```diff
@@ -0,0 +1,89 @@
+          git config user.email github-actions@github.com
+          
+      - name: Start release
+        run: |
+          git checkout develop
+          git checkout -b release/${{ needs.compute-version.outputs.package_version }}
+          
+      - name: Commit
+        run: |
+          # Change version number, commit and push
+          sed -i 's#<PackageVersion>[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]#<PackageVersion>${{ needs.compute-version.outputs.package_version }}#g' ./src/Api.Definitions/Api.Definitions.csproj
+          sed -i 's#<AssemblyVersion>[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]#<AssemblyVersion>${{ needs.compute-version.outputs.package_version }}#g' ./src/Api.Definitions/Api.Definitions.csproj
+          sed -i 's#<AssemblyVersion>[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]#<AssemblyVersion>${{ needs.compute-version.outputs.package_version }}#g' ./src/Api.Rest.Dtos/Api.Rest.Dtos.csproj
+          sed -i 's#<AssemblyVersion>[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]#<AssemblyVersion>${{ needs.compute-version.outputs.package_version }}#g' ./src/Api.Rest.Dtos/Api.Rest.Dtos.csproj
+          sed -i 's#<AssemblyVersion>[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]#<AssemblyVersion>${{ needs.compute-version.outputs.package_version }}#g' ./src/Api.Rest/Api.Rest.csproj
+          sed -i 's#<AssemblyVersion>[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]#<AssemblyVersion>${{ needs.compute-version.outputs.package_version }}#g' ./src/Api.Rest/Api.Rest.csproj
+          git add ./src/Api.Definitions/Api.Definitions.csproj
+          git add ./src/Api.Rest.Dtos/Api.Rest.Dtos.csproj
+          git add ./src/Api.Rest/Api.Rest.csproj
+          git commit -m "Raises version number to ${{ needs.compute-version.outputs.package_version }}"
+          
+      - name: Checkout master
+        run: |
+          git checkout master
+          
+      - name: Merge release into master
+        run: |
+          git merge --no-ff release/${{ needs.compute-version.outputs.package_version }}
+          
+      - name: Tag release commit
+        run: |
+          git tag v${{ needs.compute-version.outputs.package_version }}
+           
+      - name: Checkout develop
+        run: |
+          git checkout develop
+          
+      - name: Merge back master into develop
+        run: |
+          git merge --no-ff master
+          
+      - name: Push all branches
+        run: |
+          git checkout master
+          git push origin master --tags
+          git checkout develop
+          git push origin develop
+          git checkout release/${{ needs.compute-version.outputs.package_version }}
+          git push origin release/${{ needs.compute-version.outputs.package_version }}
+          
+  build-and-publish:
+    needs: release-flow
+    uses: ./.github/workflows/build-and-pack.yml
+    with:
+      configuration: "Release"
+      prerelease: "false"
+      publish_target: "nuget.org"
+    secrets: inherit
+      
```

Comment:
What kind of encoding did you use? Check that it's UTF-8 (if possible with BOM, but GitHub might have problems with that. try it).

Reply:
All pipeline files are now UTF-8 with BOM